### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sergiomons**

### CODE REVIEW

File: staff/sergiomons/hangman-app-server/design/index.html [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    <script src="vendor/jquery/3.3.1/jquery-3.3.1.min.js"></script>
+    <script src="vendor/popper/1.14.3/popper.min.js"></script>
+    <script src="vendor/bootstrap/4.1.0/js/bootstrap.min.js"></script>
+</body>
+
+</html>
+
+
+
+<!-- res.send(`<!DOCTYPE html>
```
Comment (agandia9) [inline staff/sergiomons/hangman-app-server/design/index.html]: !!

File: staff/sergiomons/hangman-app-server/views/play.pug [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                h3.secretWord Secret word
+                h2.mt-5.mb-3 #{guessed}
+        if status === 0              
+            .row.mx-auto.justify-content-center.mt-3
+                form.form-inline.my-2.my-lg-0(action="/check" method="post")
+                    input.form-control.mr-sm-2(type="text" name="text" autofocus placeholder="Enter a letter or Word")
+                    button.btn.btn-outline-info.my-2.my-sm-0(type="submit") Try
+            .row.justify-content-center
+                h4.text-center.mt-5
+                    b
```
Comment (agandia9) [inline staff/sergiomons/hangman-app-server/views/play.pug]: jeje..

File: staff/sergiomons/hangman-app-server/design/index.html (hunk 1 above)
Comment (sergiomons) [reply staff/sergiomons/hangman-app-server/design/index.html]: OK, I'll delete it!

File: staff/sergiomons/hangman-app-server/views/play.pug (hunk 2 above)
Comment (sergiomons) [reply staff/sergiomons/hangman-app-server/views/play.pug]: Yes! I am going to change it!

**Pull request merged**